packages/dashboard/src/components/contextMenu/option.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { MouseEventHandler } from 'react';
+
+import './option.css';
+
+export type ContextMenuOptionProps = {
+  disabled: boolean;
+  text: string;
+  hotkey?: string;
+  action: MouseEventHandler<Element>;
+};
+
+const ContextMenuOption: React.FC<ContextMenuOptionProps> = ({ disabled, text, hotkey, action }) => {
+  return (
+    <div
```

Comment:
Non-blocking: Could we make these minimally accessible elements instead of divs?

Reply:
👍 I'll convert these to the list notation tags

Reply:
I think they'd need to be <a> or something keyboard-navigable https://stackoverflow.com/questions/3486110/make-a-list-item-clickable-html-css

Reply:
Either that or give them tabindex

Reply:
I see, let me test that functionality.